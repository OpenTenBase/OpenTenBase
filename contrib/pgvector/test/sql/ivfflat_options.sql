SET client_min_messages = warning;
CREATE EXTENSION IF NOT EXISTS vector;
SET enable_seqscan = off;

CREATE TABLE t (val vector(3));
CREATE INDEX ON t USING ivfflat (val) WITH (lists = 0);
CREATE INDEX ON t USING ivfflat (val) WITH (lists = 32769);

SHOW ivfflat.probes;

DROP TABLE t;
