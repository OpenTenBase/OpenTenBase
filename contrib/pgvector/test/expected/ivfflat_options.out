SET client_min_messages = warning;
CREATE EXTENSION IF NOT EXISTS vector;
SET enable_seqscan = off;
CREATE TABLE t (val vector(3));
CREATE INDEX ON t USING ivfflat (val) WITH (lists = 0);
ERROR:  value 0 out of bounds for option "lists"
DETAIL:  Valid values are between "1" and "32768".
CREATE INDEX ON t USING ivfflat (val) WITH (lists = 32769);
ERROR:  value 32769 out of bounds for option "lists"
DETAIL:  Valid values are between "1" and "32768".
SHOW ivfflat.probes;
 ivfflat.probes 
----------------
 1
(1 row)

DROP TABLE t;
